mmdet/models/dense_heads/nasfcos_head.py

```diff
@@ -0,0 +1,85 @@
+import copy
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, bias_init_with_prob, Scale
+
+from mmdet.models.dense_heads.fcos_head import FCOSHead, INF
+from mmdet.ops import ModulatedDeformConvPack
+from ..builder import HEADS
+
+
+@HEADS.register_module()
+class NASFCOSHead(FCOSHead):
+    """Anchor-free head used in `NASFCOS <https://arxiv.org/abs/1906.04423>`_.
+
+    It is quite similar with FCOS head, except for the searched structure
+    of classification branch and bbox regression branch, where a structure
+    of "dconv3x3, conv3x3, dconv3x3, conv1x1"  is utilized instead.
+
+    Example:
+        >>> self = NASFCOSHead(11, 7, ["dconv3x3", "skip", "conv3x3", "skip", "dconv3x3", "conv1x1"])
```

Comment:
For the argument `arch`, could we directly pass the list of conv cfg to build the convs?